linera-core/src/tracker.rs

```diff
@@ -71,13 +71,20 @@ impl NotificationTracker {
 #[cfg(test)]
 pub mod tests {
     use super::*;
+    use linera_base::crypto::{BcsSignable, CryptoHash};
+
+    #[derive(Debug, serde::Serialize, serde::Deserialize)]
+    struct Foo;
+
+    impl BcsSignable for Foo {}
```

Comment:
`BcsHashable` ?

Reply:
Indeed it is enough, I think it was asking for BcsSignable the first time I wrote it

Reply:
for `target_hash`, it is more an origin than a destination even though it's going downward, so it's weird to put `to` or `target`, I think with the name `read_values_downward(..., from...)` it makes more sense.